docs/sdks/server-sdks/go/quickstart.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: Quickstart
+sidebar_position: 2
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Eppo Go SDK enables feature flags and experiments in your Go applications with only a few lines of code.
+
+The SDK handles all the complexity of feature flag evaluation and experiment assignment locally in your application, with no network calls required after initial setup. This guide will walk you through installing the SDK and implementing your first feature flag, experiment, and contextual bandit.
+
+## Installation
+
+First, install the SDK using go get:
+
+```bash
+go get github.com/eppo-exp/eppo-go-sdk
+```
+
+## Feature Flags
+Feature flags are a way to toggle features on and off without needing to deploy code.
+
+### Initialize the SDK
+
+[Create an SDK key](/sdks/sdk-keys) if you don't already have one. 
+
+First, initialize the SDK using your SDK key:
+
+```go
+import (
+    "github.com/eppo-exp/eppo-go-sdk/eppo"
+)
+
+config := eppo.NewConfig("<SDK-KEY>")
+client, err := eppo.Init(config)
+if err != nil {
+    log.Fatal("Failed to initialize Eppo client:", err)
+}
+```
+
+:::note
+The SDK key is different from the project API key. You can find your SDK key in the [SDK Keys section of the Eppo interface](https://eppo.cloud/configuration/environments/keys).
+:::
+
+### Assign a variant
+
+Once initialized, you can start making assignments:
+
+```go
+subjectAttributes := map[string]interface{}{
+    "country": user.Country,
+    "age":     30,
+}
+
+variation, err := client.GetStringAssignment(
+    "show-new-feature",
+    user.ID,
+    subjectAttributes,
+    "control",
+)
+if err != nil {
+    log.Printf("Error getting assignment: %v", err)
+    return
+}
+
+switch variation {
+case "variant-a":
+    handleVariantA()
+case "variant-b":
+    handleVariantB()
+default:
+    handleControl()
+}
+```
+
+### Assignment Types
+
+The SDK provides different assignment functions based on the type of value you need:
+
+| Function | Return Type |
+|----------|-------------|
+| `GetStringAssignment()` | string |
+| `GetBoolAssignment()` | bool |
+| `GetJSONAssignment()` | map[string]interface{} |
+| `GetIntAssignment()` | int |
+| `GetFloatAssignment()` | float64 |
+
+:::note
+See more details about assignment functions in the [Assignments](/sdks/server-sdks/go/assignments) page.
+:::
+
+## Experiments
+
+While feature flags are useful, they do not send you any information about how your users are interacting with the feature. Experiments provide a way to collect data about these interactions using your preferred logging system.
```

Comment:
minor: suggest differentiating between application logging and event logging. For most software developers, unqualified "logging" means application/text/debug logging
```suggestion
While feature flags are useful, they do not send you any information about how your users are interacting with the feature. Experiments provide a way to collect data about these interactions using your preferred event logging system.
```

Reply:
Thank you for that input. I'll make sure to apply this suggestion to the other SDKs if they have a similar imprecise use of logging in them